eureka/S3_data_reduction/niriss.py

```diff
@@ -459,12 +473,119 @@ def wave_NIRISS(wavefile, meta):
     return meta
 
 
-def fit_bg(data, meta):
+def fit_bg(data, meta, n_iters=3, readnoise=5, sigclip=[4,2,3], isplots=0):
     """
     Subtracts background from non-spectral regions.
 
-    # want to create some background mask to pass in to 
-      background.fitbg2
+    Parameters
+    ----------
+    data : object
+    meta : object
+    n_iters : int, optional
+       The number of iterations to go over and remove cosmic
+       rays. Default is 3.
+    readnoise : float, optional
+       An estimation of the readnoise of the detector.
+       Default is 5.
+    sigclip : list, array, optional
+       A list or array of len(n_iiters) corresponding to the
+       sigma-level which should be clipped in the cosmic
+       ray removal routine. Default is [4,2,3].
```

Comment:
Why is the default 4,2,3? I would expect something like 4,4,4.

Reply:
Yeah, I've updated this!